data/root/root.file_list

```diff
@@ -359,7 +359,11 @@ cracklib: nodeps
   x etc/ssh /lib
   x ../rescue/mount-rootfs-and-do-chroot.sh /bin
 
-openssl:
+if exists(openssl, /usr/share/ssl)
+  openssl:
```

Comment:
The directory is not on both and you get a warning logged. I wanted to avoid that.